packages/components/select/src/useSelect.ts

```diff
@@ -752,6 +752,28 @@ export const useSelect = (props: ISelectProps, emit) => {
     return Number.parseFloat(style.gap || '6px')
   }
 
+  const handleKeydown = (event: KeyboardEvent) => {
+    switch (event.key) {
+      case 'ArrowDown':
```

Comment:
Here you can use the event alias in `aria.ts` to make judgments.
https://github.com/element-plus/element-plus/blob/4c3c81e3d61ec8fba411b6a4a233de28acaec8b8/packages/constants/aria.ts#L1